cmd/checkup-launcher.go

```diff
@@ -37,6 +35,14 @@ func main() {
 		log.Fatalf("Failed to create checkup spec: %v\n", err.Error())
 	}
 
+	workspace := checkup.NewCheckupWorkspace(checkupSpec)
```

Comment:
Please consider adding the clientset to the Workspace struct.

Reply:
I prefer to leave the client "out" where possible it will make the func's that gets it as parameter easier to test 
by replacing the k8s client with with a fake client for example.